deployer/src/infra/csi.rs

```diff
@@ -0,0 +1,77 @@
+use super::*;
+use tokio::{
+    net::UnixStream,
+    time::{sleep, Duration},
+};
+use tonic::transport::{Endpoint, Uri};
+use tower::service_fn;
+
+use rpc::csi::{identity_client::IdentityClient, GetPluginInfoRequest};
+
+const CSI_SOCKET: &str = "/var/tmp/csi.sock";
+
+#[async_trait]
+impl ComponentAction for Csi {
+    fn configure(&self, options: &StartOptions, cfg: Builder) -> Result<Builder, Error> {
+        Ok(if options.no_csi {
+            cfg
+        } else {
+            if options.build {
+                std::process::Command::new("cargo")
+                    .args(&["build", "-p", "rest", "--bin", "rest"])
```

Comment:
This should build the csi controller and not rest

Reply:
Yes, indeed. Thanks !